build/build.sh

```diff
@@ -19,42 +19,15 @@ manubot process \
   --cache-directory=ci/cache \
   --log-level=INFO
 
-# pandoc settings
-CSL_PATH=build/assets/style.csl
-BIBLIOGRAPHY_PATH=output/references.json
-INPUT_PATH=output/manuscript.md
-
 # Make output directory
 mkdir -p output
 
 # Create HTML output
-# http://pandoc.org/MANUAL.html
+# https://pandoc.org/MANUAL.html
 echo >&2 "Exporting HTML manuscript"
 pandoc --verbose \
-  --from=markdown \
-  --to=html5 \
-  --filter=pandoc-fignos \
-  --filter=pandoc-eqnos \
-  --filter=pandoc-tablenos \
-  --bibliography="$BIBLIOGRAPHY_PATH" \
-  --csl="$CSL_PATH" \
-  --metadata link-citations=true \
-  --include-after-body=build/themes/default.html \
-  --include-after-body=build/plugins/anchors.html \
-  --include-after-body=build/plugins/accordion.html \
-  --include-after-body=build/plugins/tooltips.html \
-  --include-after-body=build/plugins/jump-to-first.html \
-  --include-after-body=build/plugins/link-highlight.html \
-  --include-after-body=build/plugins/table-of-contents.html \
-  --include-after-body=build/plugins/lightbox.html \
-  --include-after-body=build/plugins/attributes.html \
-  --mathjax \
-  --variable math="" \
-  --include-after-body=build/plugins/math.html \
-  --include-after-body=build/plugins/hypothesis.html \
-  --include-after-body=build/plugins/analytics.html \
-  --output=output/manuscript.html \
-  "$INPUT_PATH"
+  --defaults=build/pandoc-defaults/common.yaml \
```

Comment:
The proposal in https://github.com/manubot/rootstock/pull/261#issuecomment-568220679 would give more flexibility for specifying where the defaults files reside.

Reply:
@agitter are you suggesting that we allow setting the directory like:

```
PANDOC_DEFAULTS_DIR="${PANDOC_DEFAULTS_DIR:-build/pandoc-defaults}"
``

This means there's a single environment variable and that users could do:

```shell
cp build/pandoc-defaults build/custom-pandoc-defaults
# edit files in build/custom-pandoc-defaults
export PANDOC_DEFAULTS_DIR=build/custom-pandoc-defaults
```

Or do you think each file should have it's own variable like `PANDOC_DEFAULTS_COMMON_PATH`?

Reply:
I propose that initially we do:
```
PANDOC_DEFAULTS_DIR="${PANDOC_DEFAULTS_DIR:-build/pandoc-defaults}"
```
Your example customization above looks pretty straightforward.

That may end up being a placeholder depending what changes as part of
> work on python code to detect default files (either additional or replacement files) in the manubot package.